recipes/denoptim/LICENSE.txt

```diff
@@ -0,0 +1,619 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
This file isn't needed. License is in source archive.

Reply:
Removed in 0318ffd2a56a752434233f967dc8b0b70de302fc